torch/csrc/jit/passes/quantization.cpp

```diff
@@ -104,10 +107,29 @@ void addQuantDeQuantNodesForInput(Value* v, Node* n) {
   // Attach inputs to quant and dequant nodes
   quant->addInput(v);
   // Default Quant Params <Scale:1.0, ZeroPoint:0>
-  insertQuantNodeParams(quant, std::make_tuple(1.0, 0));
+  insertQuantNodeParams(quant, std::get<1>(qparam),
+    std::get<2>(qparam));
   dequant->addInput(quant->output());
 }
 
+template <typename... ArgT>
+bool matchQParamDictKeytoObserver(Node* n,
+  std::unordered_map<std::string, std::tuple<ArgT...>>& qparam_dict,
+  std::unordered_map<Value*, std::tuple<ArgT...>>& qparam_value_dict) {
+  if (n->kind() != prim::PythonOp) {
```

Comment:
A node might be `PythonOp`, but not an observer. In that case this will crash.

Reply:
yeah thats why  we match the second argument as confirmation it is indeed observer. Let me add more checks to ensure it does not crash